DESCRIPTION

```diff
@@ -1,67 +1,66 @@
 Package: experiencesdashboard
 Title: Patient Experiences Dashboard
 Version: 0.4.0
```

Comment:
Bump version

Reply:
what will the version be: `0.7.0` (inline with next release tag) or `0.5.0`??